src/harness/util.py

```diff
@@ -376,3 +375,22 @@ def addGrantIdsToRequests(grant_ids, requests):
     requests: (list) list of requests, containing dictionaries.
   """
   addIdsToRequests(grant_ids, requests, 'grantId')
+
+def getCertFilename(cert_name):
+  """Returns the absolute path of the file corresponding to the given |cert_name|.
+  """
+  harness_dir = os.path.dirname(os.path.abspath
```

Comment:
I am concerned about the use of absolute path in the configs. Please reconsider.

Reply:
This   method is removed. The cert and key file is received as relative path to the   files.